### DISCUSSION TITLE
Zooming function for Cartesian Chart

### DISCUSSION THREAD

**Discussion by IAmJesper** [Live-Charts/LiveCharts2#1155] [Q&A]
I'd like to implement a function which can zoom the chart in or out by pressing a button on the screen instead of just using the mouse wheel. The devices we use can have some difficulties scrolling so it would be good to be able to implement some sort of function that does this. I couldn't find a function that does this directly, but something like ZoomOnPointerView could be used to create a CartesianChart.Zoom(double XAmount, double YAmount) function or something similar.

Would this work? How would I implement this? Is there another way to do this?

**Comment by beto-rodriguez** [comment]
Hi, yes this is possible, zomming and panning is controlled by the [MinLimit](https://lvcharts.com/api/2.0.0-beta.900/LiveChartsCore.Axis-3#property-MinLimit) and [MaxLimit](https://lvcharts.com/api/2.0.0-beta.900/LiveChartsCore.Axis-3#property-MaxLimit) properties, you can find a related example at the [paging sample](https://lvcharts.com/docs/WPF/2.0.0-beta.900/samples.axes.paging).
